test/stdlib/TestCharacterSet.swift

```diff
@@ -192,6 +192,22 @@ class TestCharacterSet : TestCharacterSetSuper {
         expectEqual(expected, union)
     }
 
+    func test_subtractAndFormSymmetricDifference() {
+        var set1 = CharacterSet(charactersIn: "abc")
+        let set2 = CharacterSet(charactersIn: "b")
+        set1.subtract(set2)
+        expectFalse(set1.contains("b"))
+        set1.formSymmetricDifference(set2)
```

Comment:
I've refactored as requested so that there's only one assertion per test, so this here reuse of `set1` is gone.